src/dsp_tools/xmllib/value_converters.py

```diff
@@ -18,3 +21,57 @@ def convert_to_bool_string(value: Any) -> str:
     elif str_val in ("true", "1", "1.0", "yes"):
         return "true"
     return str(value)
+
+
+def replace_newlines_with_tags(text: str, converter_option: NewlineReplacementTags) -> str:
+    """
+    Converts the newlines in a string to XML tags.
+    The type of tags is specified through the converter_option enum.
+
+    Args:
+        text: string to convert
+        converter_option: tag options
+
+    Returns:
+        string with replaced values
+
+    Raises:
+        InputError: If an invalid conversion option is given
+    """
+    match converter_option:
+        case NewlineReplacementTags.none:
+            return text
+        case NewlineReplacementTags.newline:
+            return replace_newlines_with_br_tags(text)
+        case NewlineReplacementTags.paragraph:
+            return replace_newlines_with_paragraph_tags(text)
+        case _:
+            raise InputError(f"The conversion option for the text does not exist: {converter_option}.")
+
+
+def replace_newlines_with_paragraph_tags(text: str) -> str:
```

Comment:
@BalduinLandolt 
Ich erhalte hier ein linter problem:

./src/dsp_tools/xmllib/value_converters.py:52 (replace_newlines_with_paragraph_tags): DAR201 Missing "Returns" in Docstring: - return
./src/dsp_tools/xmllib/value_converters.py:52 (replace_newlines_with_paragraph_tags): DAR101 Missing parameter(s) in Docstring: - text
./src/dsp_tools/xmllib/value_converters.py:67 (replace_newlines_with_br_tags): DAR201 Missing "Returns" in Docstring: - return
./src/dsp_tools/xmllib/value_converters.py:67 (replace_newlines_with_br_tags): DAR101 Missing parameter(s) in Docstring: - text

Ich sehe nicht was ich am docstring anders machen kann. Siehst du den Fehler? Danke!

Reply:
Darglint is complaining because the docstring doesn't follow the syntax rules for reStructuredText (rst). rst is the language in which the docstrings are written. 

There are 2 problems here:
- the `\` is not escaped, it should be `\\`
- HTML/XML tags are not surrounded by backticks, so they are interpreted as markup. So they are wrongly rendered: 
  <img width="400" alt="Screenshot 2024-10-07 at 11 30 18" src="https://github.com/user-attachments/assets/546d3c82-cd1e-499e-bacd-59c6b2ed481c">

I tried it out, if you fix these 2 things, it will work.

Read more about rst: 
- https://docutils.sourceforge.io/rst.html
- https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html